grunt-configs/requirejs.js

```diff
@@ -155,14 +155,12 @@ module.exports = function(grunt, options) {
                 name: 'bootstraps/video-player',
                 out: options.staticTargetDir + 'javascripts/bootstraps/video-player.js',
                 paths: {
```

Comment:
sounds like a good idea to move this into the shared options paths object, due to the lack of support for merging paths references. Will save a lot of developers debugging time.